demosplan/DemosPlanCoreBundle/Logic/Rpc/RpcErrorGenerator.php

```diff
@@ -16,27 +16,6 @@
 
 class RpcErrorGenerator implements RpcErrorGeneratorInterface
 {
-    final public const ACCESS_DENIED_CODE = -32768;
-    final public const ACCESS_DENIED_MESSAGE = 'Access denied';
-
-    final public const INTERNAL_ERROR_CODE = -32603;
-    final public const INTERNAL_ERROR_MESSAGE = 'Internal error';
-
-    final public const INVALID_PARAMS_CODE = -32602;
-    final public const INVALID_PARAMS_MESSAGE = 'Invalid params';
-
-    final public const INVALID_REQUEST_CODE = -32600;
-    final public const INVALID_REQUEST_MESSAGE = 'Invalid Request';
-
-    final public const METHOD_NOT_FOUND_CODE = -32601;
-    final public const METHOD_NOT_FOUND_MESSAGE = 'Method not found';
-
-    final public const PARSE_ERROR_CODE = -32700;
-    final public const PARSE_ERROR_MESSAGE = 'Parse error';
-
-    final public const SERVER_ERROR_CODE = -32000;
-    final public const SERVER_ERROR_MESSAGE = 'Server error';
```

Comment:
If you move these to demosplan-addon I would expect you have to also change the places where the codes are used, which I don't see here or in the other PRs.

Reply:
done in d7787946d844b3d59c570a69083f06f3e0f1fdb4

Reply:
and also forgot about adding the interface to the services.yml 43f67e5edc66c71f0d5e7470d86485d8bf5afaeb